src/components/tooltip/Tooltip.tsx

```diff
@@ -274,16 +282,32 @@ class Tooltip extends React.Component<TooltipProps, State> {
         return (
             <TetherComponent ref={this.tetherRef} {...tetherProps}>
                 {React.cloneElement(React.Children.only(children) as React.ReactElement, componentProps)}
-                {showTooltip && (
-                    <div className={classes} id={this.tooltipID} role="tooltip" aria-live="polite">
-                        {text}
-                        {withCloseButton && (
-                            <PlainButton className="tooltip-close-button" onClick={this.closeTooltip}>
-                                <IconClose className="bdl-Tooltip-iconClose" width={14} height={14} />
-                            </PlainButton>
-                        )}
-                    </div>
-                )}
+                {showTooltip &&
+                    (shouldStopEventPropagation ? (
+                        <div className={classes} id={this.tooltipID} role="tooltip" aria-live="polite">
+                            <div
+                                role="presentation"
+                                onClick={this.handleTooltipEvent}
+                                onKeyPress={this.handleTooltipEvent}
+                            >
+                                {text}
```

Comment:
It looks like the component has been duplicated. Can we move this to a separate method? Or move the wrapper to a separate component which returns a Fragment if the prop isn't enabled?

Reply:
will fix